src/main/java/org/spongepowered/api/effect/Viewer.java

```diff
@@ -161,4 +161,60 @@ default void resetBlockChange(final Vector3i position) {
      */
     void resetBlockChange(int x, int y, int z);
 
+    /**
+     * Sends a client-only block breaking progress.
+     *
+     * <p>In vanilla breaking progress will not be rendered if player
+     * is further than 32 blocks from given position.<br>
+     * Sent breaking progress expires on client 400 ticks after receiving.</p>
+     *
+     * @param position The position
+     * @param progress The breaking progress from 0 to 1 (1 excluded)
+     */
+    default void sendBlockProgress(final Vector3i position, final float progress) {
```

Comment:
Should we use double instead of float?

Reply:
Maybe just for comfort so you could do 0.1 instead of 0.1F